World/Updates/Rel21/Rel21_22_022_Q7636_P2_FIX_Artorius_the_Doombringer.sql

```diff
@@ -0,0 +1,291 @@
+		(14531,103,7498.49,-4587.74,617.458,6.247,0,0),
+		(14531,104,7511.17,-4598.02,624.285,4.777,0,0),
+		(14531,105,7511.59,-4604.58,626.979,4.777,0,0),
+		(14531,106,7512.09,-4612.2,631.305,4.777,0,0),
+		(14531,107,7512.09,-4612.2,631.305,4.777,0,0),
+		(14531,108,7512.59,-4619.81,636.06,5.368,0,0),
+		(14531,109,7515.91,-4624.14,639.219,5.368,0,0),
+		(14531,110,7524.58,-4635.4,644.969,5.368,0,0),
+		(14531,111,7524.58,-4635.4,644.969,0.558,0,0),
+		(14531,112,7531.11,-4631.32,642.777,0.558,0,0),
+		(14531,113,7539.48,-4626.1,640.701,0.806,0,0),
+		(14531,114,7550.04,-4615.08,638.415,0.806,0,0),
+		(14531,115,7563.5,-4604.11,636.139,0.561,0,0),
+		(14531,116,7563.5,-4604.11,636.139,0.561,0,0),
+		(14531,117,7586.61,-4591.86,636.012,0.413,0,0),
+		(14531,118,7592.64,-4589.21,637.272,0.413,0,0),
+		(14531,119,7600.65,-4585.69,637.973,0.413,0,0),
+		(14531,120,7600.65,-4585.69,637.973,0.413,0,0),
+		(14531,121,7611.67,-4580.85,639.924,0.413,0,0),
+		(14531,122,7630.22,-4574.36,645.151,0.215,0,0),
+		(14531,123,7630.22,-4574.36,645.151,0.215,0,0),
+		(14531,124,7639.86,-4572.25,648.197,0.41,0,0),
+		(14531,125,7644.93,-4570.04,649.304,0.41,0,0),
+		(14531,126,7665.32,-4562.1,652.39,0.313,0,0),
+		(14531,127,7665.32,-4562.1,652.39,0.313,0,0),
+		(14531,128,7684.39,-4557.87,654.888,0.068,0,0),
+		(14531,129,7684.39,-4557.87,654.888,0.068,0,0),
+		(14531,130,7705.13,-4556.45,659.16,0.068,0,0),
+		(14531,131,7705.13,-4556.45,659.16,0.068,0,0),
+		(14531,132,7724.7,-4555.12,664.672,0.068,0,0),
+		(14531,133,7730.69,-4559.84,667.699,5.616,0,0),
+		(14531,134,7738.45,-4565.94,671.137,5.616,0,0),
+		(14531,135,7738.45,-4565.94,671.137,5.616,0,0),
+		(14531,136,7746.2,-4572.05,673.5,5.616,0,0),
+		(14531,137,7754.78,-4578.8,675.948,5.616,0,0),
+		(14531,138,7762.46,-4584.85,678.841,5.616,0,0),
+		(14531,139,7769.33,-4590.26,682.134,5.616,0,0),
+		(14531,140,7778.79,-4597.71,686.727,5.616,0,0),
+		(14531,141,7785.67,-4603.12,689.915,5.616,0,0),
+		(14531,142,7793.42,-4609.23,693.801,5.616,0,0),
+		(14531,143,7807.44,-4619.7,700.296,5.666,0,0),
+		(14531,144,7815.39,-4625.34,703.078,5.666,0,0),
+		(14531,145,7815.39,-4625.34,703.078,5.666,0,0),
+		(14531,146,7823.38,-4631.01,706.286,5.861,0,0),
+		(14531,147,7831.36,-4634.59,709.327,5.861,0,0),
+		(14531,148,7841.33,-4639.06,713.162,0.021,0,0),
+		(14531,149,7850.05,-4638.88,714.439,0.021,0,0),
+		(14531,150,7868.27,-4633.71,714.233,0.599,0,0),
+		(14531,151,7875.49,-4628.78,713.59,0.599,0,0),
+		(14531,152,7883.65,-4623.21,712.769,0.599,0,0),
+		(14531,153,7891.8,-4617.65,711.935,0.599,0,0),
+		(14531,154,7898.69,-4610.59,710.972,0.796,0,0),
+		(14531,155,7904.81,-4604.33,710.23,0.796,0,0),
+		(14531,156,7909.45,-4599.58,710.154,0.796,0,0);
+		
+		
+		/* Fix translation for emote of Solenor the slayer */
+		/* It was forgotten in the last commit related to this npc  */
+		UPDATE `script_texts` SET `content_loc2` = '%s est immobilisé.' WHERE `entry` = -1001252;
```

Comment:
These should not be here and needs to be part of the localisation repo instead.
- Ideally added via MAGNET